khal/khalendar/event.py

```diff
@@ -311,7 +311,9 @@ def _create_calendar():
         """
         calendar = icalendar.Calendar()
         calendar.add('version', '2.0')
-        calendar.add('prodid', '-//CALENDARSERVER.ORG//NONSGML Version 1//EN')
+        calendar.add(
+            'prodid', '-//PIMUTILS.ORG//NONSGML khal / icalendar //EN'
```

Comment:
Non Standard Generalized Markup Language, we could really put anything in there, but somehow everybody seems to use this format.